front/src/store/index.js

```diff
@@ -123,9 +123,19 @@ export default new Vuex.Store({
                                 user.email,
                                 user.password
                             )
-                            .then(() => {
-                                commit('setStatus', 'success');
-                                commit('setError', null);
+                            .then(registered => {
+                                registered.user.updateProfile({
+                                    displayName: `${user.firstName} ${user.lastName}`
+                                })
```

Comment:
```suggestion
                                return registered.user.updateProfile({
                                    displayName: `${user.firstName} ${user.lastName}`
                                })
```

And move this before final catch ?
```
.then(() => {
    commit('setStatus', 'success');
    commit('setError', null);
})
```